aiida/cmdline/params/options/__init__.py

```diff
@@ -430,11 +430,19 @@ def decorator(command):
 HOSTNAME = OverridableOption('-H', '--hostname', type=types.HostnameType(), help='Hostname.')
 
 TRANSPORT = OverridableOption(
-    '-T', '--transport', type=types.PluginParamType(group='transports'), required=True, help='Transport type.'
+    '-T',
+    '--transport',
+    type=types.PluginParamType(group='transports'),
+    required=True,
+    help='Transport plugin (use "?" to list available plugins).'
```

Comment:
@ltalirz added these changes to the help text, so he can defended them 😉 
I'm personally happy to revert

Reply:
This was merged in from the previous additional descriptions of the cli options in the docs.
I agree to remove these, also given the fact that this will not work in non-interactive mode